package.json

```diff
@@ -30,7 +30,7 @@
   },
   "scripts": {
     "calculate-hashes:check": "ts-node scripts/calculate-hashes.ts --check-only",
-    "calculate-hashes:fix": "ts-node scripts/calculate-hashes.ts",
+    "calculate-hashes:fix": "./recompute_hashes.sh",
```

Comment:
I think this is used in CI and recompute_hashes is very slow as it recomplies everything. Do you think it would make sense to have fix and rebuild-and-fix as recompute_hashes.sh?